templates/bundles/DemosPlanCoreBundle/DemosPlanDocument/includes/single_document_file_upload.html.twig

```diff
@@ -14,7 +15,8 @@
         name="r_document"
         needs-hidden-input
         :required="{{ required ? 'true' : 'false' }}"
-        :translations="{ dropHereOr: Translator.trans('form.button.upload.file', { browse: '{browse}', maxUploadSize: '2GB' }) }">
+        :translations="{ dropHereOr: Translator.trans('form.button.upload.file', { browse: '{browse}', maxUploadSize: '2GB' }) }"
+        :tus-endpoint="dplan.paths.tusEndpoint">
     </dp-upload-files>
 {% else %}
     <dp-upload-files
```

Comment:
This instance also needs adapting to the new props.

Reply:
Done in https://github.com/demos-europe/demosplan-core/pull/1982/commits/06690cfd644a1a49384b10076c9645f2ea5c7c0f